### DISCUSSION TITLE
Backup Phrase

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#846] [Help]
One of my user is having this problem and i cannot find the work around to solve this issue

When my user tries to login using web3auth + auth0, app.openlogin.com is asking to Verify the login by giving the backup recovery phase. Now, the issue is, my user do not have the backup recovery phase. He has lost it. Is there any way to recover the account?

**Comment by BboyStatix** [comment]
Does he have other shares e.g. device share? Could you ask him to log in with his other device? More information would be helpful.

**Reply by vaibonfire** [reply]
I have the same issue as well. Is there any method/function to let the user reset the phrase or request the backup phrase by email? The user is stuck not having access to device share and backup phrase.

**Comment by himanshuchawla009** [comment]
User must be prompted to send the backup phrase to an email since it is impossible to skip that step during mfa registration. Can you confirm with user to check email inbox for recovery phrase.
